package.json

```diff
@@ -25,20 +25,12 @@
     "prettier": "^2.0.5"
   },
   "resolutions": {
-    "**/**/axe-core": "^3.5.3",
-    "**/**/browserslist": "^4.17.0",
-    "**/**/css-what": "^5.0.1",
-    "**/**/glob-parent": "^5.1.2",
-    "**/**/json-schema": "^0.4.0",
-    "**/**/jszip": "^3.7.1",
-    "**/**/normalize-url": "^6.0.1",
-    "**/**/postcss": "^8.2.15",
-    "**/**/postcss-scss": "^3.0.5",
-    "**/**/tar": "^6.1.11",
-    "**/**/trim": "^0.0.3",
-    "**/**/url-parse": "^1.5.3",
-    "**/**/yup": "^0.31.1",
```

Comment:
Not sure why yup was pinned here (it's a dependency only of `@defencedigital/react-component-library`), but couldn't see anything wrong with it updated.